codegen/Cargo.toml

```diff
@@ -13,8 +13,9 @@ description = "Generate an API for interacting with a substrate node from FRAME
 
 [features]
 default = []
-fetch-metadata = ["dep:jsonrpsee", "dep:tokio", "dep:frame-metadata"]
+fetch-metadata = []
```

Comment:
DQ: Looking at this now, what is the reason to add a new `jsonrpsee` feature flag here, when `fetch-metadata` always requires `jsonrpsee` anyway I think?

Reply:
It's possible to fetch-metadata from a file without jsonrpsee that's why I kept it...

Reply:
Oohhh I missed that, hrmph! :D